Bottom: 95b3371cf42d85777be8b8a1952531e9eb2084fb
Top:    a65c58af2be11f089e4be7689b4f15a38dcf07de
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-06 17:08:21 +0000

Refresh of wip-on-quad-edge-play

---

diff --git a/src/hid/gtk/gtkhid-gl.c b/src/hid/gtk/gtkhid-gl.c
index b9830b8..1aecd0a 100644
--- a/src/hid/gtk/gtkhid-gl.c
+++ b/src/hid/gtk/gtkhid-gl.c
@@ -17,8 +17,8 @@
 
 #include "hid/common/quad.h"
 #include "hid/common/vertex3d.h"
-#include "hid/common/face3d.h"
 #include "hid/common/appearance.h"
+#include "hid/common/face3d.h"
 #include "hid/common/object3d.h"
 
 #ifndef WIN32
